src/backend/dx12/src/command.rs

```diff
@@ -980,7 +980,10 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
         unsafe { self.raw.Close(); }
     }
 
-    fn reset(&mut self, _release_resources: bool) {
+    fn reset(&mut self, release_resources: bool) {
+        if release_resources {
+            unsafe { self.allocator.Reset(); }
```

Comment:
- `release_resource` should destroy the allocator if possible as it's the only way in dx12 to regain the memory allocated.
- else we reset the allocator, which should be always valid as in the case of indiviual reset we kind have to assign an own pool to each commandbuffer

Reply:
> release_resource should destroy the allocator if possible as it's the only way in dx12 to regain the memory allocated.

What conditions would allow for that safely? `allocator.clone().as_unknown().Release() == 1`?